Makefile

```diff
@@ -34,30 +38,38 @@ bazel-bin/src/main/auth_server:
     # Note: add --compilation_mode=dbg to the end of the next line to build a debug executable with `make docker-from-scratch`
 	bazel build $(BAZEL_FLAGS) //src/main:auth_server
 
+# Always use amd64 for bazelisk for build and test rules below, since we don't support for macOS
+# arm64 (with --host_javabase=@local_jdk//:jdk) yet (especially the protoc-gen-validate project:
+# "no matching toolchains found for types @io_bazel_rules_go//go:toolchain").
+bazel := GOARCH=amd64 go run $(bazelisk@v)
+
 build:
-	bazel build $(BAZEL_FLAGS) //src/...
+	$(bazel) build $(BAZEL_FLAGS) //src/...
 
 run:
-	bazel run $(BAZEL_FLAGS) $(TARGET)
+	$(bazel) run $(BAZEL_FLAGS) $(TARGET)
 
+TEST_FLAGS ?= --strategy=TestRunner=standalone --test_output=all
 test:
-	bazel test $(BAZEL_FLAGS) --strategy=TestRunner=standalone --test_output=all --cache_test_results=no //test/...
+	$(bazel) test $(BAZEL_FLAGS) $(TEST_FLAGS) //test/...
 
 # Only run tests whose name matches a filter
 # Usage examples:
 #   make filter-test FILTER=*RetrieveToken*
 #   make filter-test FILTER=OidcFilterTest.*
 filter-test:
-	bazel test $(BAZEL_FLAGS) --strategy=TestRunner=standalone --test_output=all --cache_test_results=no //test/... --test_arg='--gtest_filter=$(FILTER)'
+	$(bazel) test $(BAZEL_FLAGS) $(TEST_FLAGS) //test/... --test_arg='--gtest_filter=$(FILTER)'
 
 coverage:
-	bazel coverage $(BAZEL_FLAGS) --instrumentation_filter=//src/ //...
+	$(bazel) coverage $(BAZEL_FLAGS) --instrumentation_filter=//src/ //...
 
+buildifier@v := github.com/bazelbuild/buildtools/buildifier@4.2.5
```

Comment:
Should we define `buildifier@v` here, even we already have had in `Tools.mk'?

Reply:
Thanks for this. I should remove this.